### DISCUSSION TITLE
Remove the translation/comment/edit menu from each post?

### DISCUSSION THREAD

**Discussion by stef41** [hbstack/.github#55] [Q&A] [answered]
![Screenshot from 2024-05-12 11-31-25](https://github.com/hbstack/.github/assets/87280280/551bd46c-f379-4bb7-92a4-023dffd670b0)
Hello how can I remove this menu from the whole website?

**Comment by razonyang** [comment, marked as answer]
You'll need to remove the `github.com/hbstack/blog/modules/content-panel` and `github.com/hbstack/docs/modules/content-panel` from configuration.
